core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -368,11 +368,28 @@ sealed abstract class IO[+E, +A] { self =>
   final def zipWith[E1 >: E, B, C](that: IO[E1, B])(f: (A, B) => C): IO[E1, C] =
     self.flatMap(a => that.map(b => f(a, b)))
 
+  /**
+   * Sequentially zips this effect with the specified effect, combining the
+   * results into a tuple.
+   */
+  final def zip[E1 >: E, B](that: IO[E1, B]): IO[E1, (A, B)] =
```

Comment:
shouldn't we call it `seq` (as a dual to `par`)?

Reply:
Good idea, in which case we should rename `zipWith` => `seqWith`, and maybe add `parWith` for symmetry.